### PR TITLE
Version 1.3.0 Release

### PR DESCRIPTION
**Pull request by SebastienGllmt**
- New "about Yoroi" tab in the settings
- Adds new link in footer on add wallet page (about hardware wallets)
- Adds ada Remeption
- Adds transfer from Daedalus paper wallet option

### CODE REVIEW

**Comment by PavelHaliakevich** [comment]
The button names in the bottom menu are not displayed in full screen mode (office monitor, 19'' 1440 x 900)
image

**Comment by PavelHaliakevich** [comment]
"Transfer all funds from Daedalus paper" gives an error message, when paper wallet is not empty but with less funds than fees (<0.16 ADA)

[2019-02-18T16:34:59+03:00] daedalusTransfer::generateTransferTx {
  "message": "NotEnoughInput",
  "name": "CardanoCryptoError",
  "stack": "CardanoCryptoError: NotEnoughInput\n    at n (chrome-extension://fjhimcbkcandmcgiohlbnokclbopoamb/js/yoroi.bundle.js:1:156604)\n    at chrome-extension://fjhimcbkcandmcgiohlbnokclbopoamb/js/yoroi.bundle.js:1:906942\n    at n (chrome-extension://fjhimcbkcandmcgiohlbnokclbopoamb/js/yoroi.bundle.js:1:1302158)\n    at Generator._invoke (chrome-extension://fjhimcbkcandmcgiohlbnokclbopoamb/js/yoroi.bundle.js:1:1303238)\n    at Generator.e.(anonymous function) [as next] (chrome-extension://fjhimcbkcandmcgiohlbnokclbopoamb/js/yoroi.bundle.js:1:1302337)\n    at n (chrome-extension://fjhimcbkcandmcgiohlbnokclbopoamb/js/yoroi.bundle.js:1:12378)\n    at i (chrome-extension://fjhimcbkcandmcgiohlbnokclbopoamb/js/yoroi.bundle.js:1:12549)"
}
[2019-02-18T16:34:59+03:00] DaedalusTransferStore::setupTransferFunds {
  "message": "api.errors.generateTransferTxError: {}",
  "name": "GenerateTransferTxError",
  "stack": "GenerateTransferTxError: api.errors.generateTransferTxError: {}\n    at chrome-extension://fjhimcbkcandmcgiohlbnokclbopoamb/js/yoroi.bundle.js:1:907350\n    at n (chrome-extension://fjhimcbkcandmcgiohlbnokclbopoamb/js/yoroi.bundle.js:1:1302158)\n    at Generator._invoke (chrome-extension://fjhimcbkcandmcgiohlbnokclbopoamb/js/yoroi.bundle.js:1:1303238)\n    at Generator.e.(anonymous function) [as next] (chrome-extension://fjhimcbkcandmcgiohlbnokclbopoamb/js/yoroi.bundle.js:1:1302337)\n    at n (chrome-extension://fjhimcbkcandmcgiohlbnokclbopoamb/js/yoroi.bundle.js:1:12378)\n    at i (chrome-extension://fjhimcbkcandmcgiohlbnokclbopoamb/js/yoroi.bundle.js:1:12549)",
  "id": "api.errors.generateTransferTxError",
  "defaultMessage": "!!!Error while generating transfer transacion.",
  "values": {}
}
image
********
I previously reported a similar issue:
https://emurgo.slack.com/archives/GC9AM0E8K/p1548260746040500

**Comment by PavelHaliakevich** [comment]
Link to Youtube from the About page leads to video that has been deleted
https://www.youtube.com/watch?v=GLNgpr-3t2E&list=PLFLTrdAG7xRZUmi04s44T1VEF20xKquF2

image

**Comment by PavelHaliakevich** [comment]
> The button names in the bottom menu are not displayed in "maximize" mode (office monitor, 19'' 1440 x 900)

Same problem with "About page" icon names.
image

File: app/components/settings/categories/AboutYoroiSettings.js [hunk 1]
```diff
@@ -119,7 +119,7 @@ export default class AboutYoroiSettings extends Component {
             message={messages.aboutYoroiFacebook}
           />
           <FooterItem
-            url="https://www.youtube.com/watch?v=GLNgpr-3t2E&list=PLFLTrdAG7xRZUmi04s44T1VEF20xKquF2"
+            url="https://www.youtube.com/channel/UCgFQ0hHuPO1QDcyP6t9KZTQ"
```
Comment (vantuz-subhuman) [inline app/components/settings/categories/AboutYoroiSettings.js]: The idea was to send people to vids specifically about Yoroi

**Comment by PavelHaliakevich** [comment]
Just in case - don't forget to add this to all Language versions :)
image

**Pull request merged**